### ISSUE TITLE
Configure `Vue.config` through plugin options

### ISSUE THREAD

**Issue by ovlb** [11ty/eleventy-plugin-vue#39]
In 1.0 we can merge `rollupOptions`. It would be nice to also merge options of the Vue config. 

11ty Vue already uses `globalProperties` ([source](https://github.com/11ty/eleventy-plugin-vue/blob/master/EleventyVue.js#L528)).

As far as I can see this there is no way to do this via Rollup and the Vue config is not exposed? 

I hope that this will allow me to add web components that should be ignored by Vue/11ty to be used in the client. If there are other ways to achieve this, I’m all ears.

**Comment by wommy** [comment]
i dont know what you need to do,  
but i was just looking over the package,  
and i was wondering if you could use [this](https://github.com/11ty/eleventy-plugin-vue/blob/master/.eleventy.js#L20-L21) somehow?

**Comment by ovlb** [comment]
@wommy I [already use this](https://github.com/inframanufaktur/no-11/blob/main/_plugins/vue.js) to change how the app gets compiled. 
Nonetheless, the [Vue app config](https://v3.vuejs.org/api/application-config.html) is a step before this. It gives Vue additional context. E.g. sets global prop (as linked in the OP,  Eleventy’s `this.page` is made accessible to the Vue components).
Another use case I’ve found where a deeper level of access to the setup of the Vue app is useful are [global components](https://dev.to/alexc957/how-to-use-global-components-in-vuejs-154m). 

In short: controlling the output is wonderful, but having better control of the input will be marvellous.
